test/ibmq/test_ibmq_client_v2.py

```diff
@@ -298,3 +231,163 @@ def test_url_unreachable(self, qe_token, qe_url):
+
+        # TODO
+        pass
+
+    @skip
+    # TODO: RequestsApiError: 404 Client Error: Not Found for url: [...] None, Error code: NotFound.
+    def test_job_download_qobj_object_storage(self):
+        """Test job Qobj download using object storage."""
+        # result = self.client.job_download_qobj_object_storage(self.job_id)
+
+        # TODO
+        pass
+
+    @skip
+    # TODO: RequestsApiError: 404 Client Error: Not Found for url: [...] None, Error code: NotFound.
+    def test_job_result_object_storage(self):
+        """Test getting a job's result using object storage."""
+        # result = self.client.job_result_object_storage(self.job_id)
+
+        # TODO
+        pass
+
+    def test_job_get_responsive(self):
+        """Test getting a job responds with something other than None."""
+        result = self.client.job_get(self.job_id)
+        status_present = ['status' in [dict_ for dict_ in result.get('qasms')]]
+        self.assertTrue(status_present)
+
+    def test_job_get_filtered_fields(self):
+        """Test getting a job responds uses field filters correctly."""
+        no_params_result = self.client.job_get(self.job_id)
+
+        # test exclusion returns a different result than the default
+        excluded_fields = ['qasms']
+        excluded_fields_result = self.client.job_get(self.job_id, excluded_fields=excluded_fields)
+        self.assertNotEqual(excluded_fields_result, no_params_result)
+
+        # test inclusion returns a different result than the default
+        included_fields = ['qasms']
+        included_fields_result = self.client.job_get(self.job_id, included_fields=included_fields)
+        self.assertNotEqual(included_fields_result, no_params_result)
+
+        # test sending both an include and exclude list uses only the include list
+        include_and_exclude_result = self.client.job_get(self.job_id,
+                                                         included_fields=included_fields,
+                                                         excluded_fields=excluded_fields)
+        self.assertDictEqual(include_and_exclude_result, included_fields_result)
+
+    def test_job_status(self):
+        """Test getting job status."""
+        result = self.client.job_status(self.job_id)
+        status_present = 'status' in result
+        self.assertTrue(status_present)
+
+    def test_job_final_status_websocket(self):
+        """Test getting a job's final status via websocket."""
+        result = self.client.job_final_status_websocket(self.job_id)
+        status_present = 'status' in result
+        self.assertTrue(status_present)
+
+    def test_job_properties(self):
+        """Test getting job properties."""
+        # TODO - is {} an acceptable response here?
+        result = self.client.job_properties(self.job_id)
+        self.assertIsNotNone(result)
+
+    # TODO: RequestsApiError: 404 Client Error: Not Found for url: [...]
+    def test_job_cancel(self):
```

Comment:
Can you remove this test? It is also one of those that require slightly trick logic (in this case, you need the job to be in a "cancellable" state) and better of included during #189